Dockerfile

```
@@ -53,30 +54,32 @@ RUN node scripts/prepare-production-manifest.js \
 # These Go binaries are NOT needed at runtime and are NOT FIPS compliant
 RUN rm -rf node_modules/esbuild node_modules/@esbuild node_modules/.bin/esbuild
 
-FROM ${BASE_IMAGE} as runtime
+FROM ${MINIMAL_IMAGE} AS runtime
 
 # The curl binary is required in the final image, as it's used for
 # liveness and readiness probes
 USER root
-RUN dnf install -y curl-minimal && dnf clean all && curl --version
+RUN microdnf install -y curl-minimal && microdnf clean all && curl --version
```

Hadolint (2.14.0)

[warning] Specify version with `dnf install -y <package>-<version>`.

(DL3041)